test/parallel/test-internal-module-map-asserts.js

```diff
@@ -21,7 +21,7 @@ const ModuleMap = require('internal/modules/esm/module_map');
   // but I think it's useless, and was not simple to mock...
   const job = undefined;
 
-  [{}, [], true, 1, () => {}].forEach((value) => {
```

Comment:
This change is necessary because the module job registry was changed to contain both jobs and functions of jobs. The functions of jobs case is necessary for CJS injection into the registry not to create early promises which would otherwise introduce noise into the async hooks queue.